ci-test.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+if [[ "${NEAR_RELEASE}" == "true" ]]; then
+    echo "Test with release version of borsh and near-vm-logic"
+    sed -n '/^borsh/p' near-bindgen/Cargo.toml 
+    sed -n '/^near-vm-logic/p' near-bindgen/Cargo.toml
+    cargo test --all
+else
+    echo "Test with git version of borsh and near-vm-logic"
+
+    cp near-bindgen/Cargo.toml{,.bak}
+    cp Cargo.lock{,.bak}
+    borsh_git='borsh = { git = "https://github.com/nearprotocol/borsh", branch = "master" }'
```

Comment:
I think using commented out `[patch]` is more maintainable because it requires only one sed and people can also test it manually on their machine if they see CI failing.

Reply:
Make sense